app/models/customer.rb

```diff
@@ -0,0 +1,10 @@
+class Customer < ActiveRecord::Base
+	has_many :orders
+	validate :custom_validation
+
+	def custom_validation
+		errors.add(:name, "You must enter name") if name.blank?
```

Comment:
@amaninquirly  Custom validation needed only when if you need to handle other than the default validate function. The above case you can handle using **validate_presence_of**.